source/includes/_enrichment.md

```diff
@@ -1077,6 +1078,7 @@ hashed_email (optional) | The person's md5 or sha256 hashed email | 8d935115b9ff
 organization_name (optional) | The person's company name | Apollo Inc.
 domain (optional) | The person's company domain | apollo.io
 id (optional) |  The person's ID obtained from the search endpoint | "583f2f7ed9ced98ab5bfXXXX"
+linkedin_url (optional) | The person's linkedin URL | http://www.linkedin.com/in/tim-zheng
```

Comment:
Can you add this param in the request body of the sample example given in shell and python for the customers to know how they need to pass linkedin_url?